WISE-Community/WISE-Client
# Added before/after scores and change in score column to Milestone report Student Work displays

To test:
- Verify that milestones with multiple locations show "Initial", "Revised", and "+/-" score columns when viewing Student Work.
- Make sure the before, after, and change in score values are calculated correctly for each workgroup. (Before score will match the workgroup's score for the first milestone location, after score will match the last location, change in score will equal after - before.
- Verify that you can sort workgroups by each of the columns.
- Verify that milestones with only one location work as before.

Resolves #13.
